assets/model_monitoring/components/src/model_monitor_output_metrics/entities/signals/data_quality_signal.py

```diff
@@ -139,10 +139,11 @@ def _build_feature_metrics(
 
             # Add the feature importance metrics
             if len(featureimportance_dictionary) != 0:
-                feature_importance_metric = {
-                    "metricName": "BaselineFeatureImportance",
-                    "metricValue": featureimportance_dictionary[feature_name],
-                }
-                output[feature_name].metrics.append(feature_importance_metric)
+                if not any('BaselineFeatureImportance' in d.values() for d in output[feature_name].metrics):
```

Comment:
you also need this change for data drift, right?

Reply:
no. In data quality we have multiple signal metrics for single feature .. data drift have feature based signal

Reply:
Added same check in Datadrift too